papis/commands/addto.py

```diff
@@ -95,22 +128,18 @@ def run(document: papis.document.Document,
 @papis.cli.query_argument()
 @papis.cli.git_option(help="Add and commit files")
 @papis.cli.sort_option()
-@click.option(
-    "-f", "--files",
-    help="File fullpaths to documents",
-    multiple=True,
-    type=click.Path(exists=True))
-@click.option(
-    "--file-name",
-    help="File name for the document (papis format)",
-    default=None)
+@click.option("-f",
+              "--files",
+              help="File fullpaths to documents",
+              multiple=True,
+              type=click.Path(exists=True))
+@click.option("-u", "--urls", help="URLs to documents", multiple=True)
+@click.option("--file-name",
+              help="File name for the document (papis format)",
+              default=None)
 @papis.cli.doc_folder_option()
-def cli(query: str,
-        git: bool,
-        files: List[str],
-        file_name: Optional[str],
-        sort_field: Optional[str],
-        doc_folder: str,
+def cli(query: str, git: bool, files: List[str], urls: List[str],
+        file_name: Optional[str], sort_field: Optional[str], doc_folder: str,
```

Comment:
Can you leave these as one per line? I think most of the commands are formatted like that, so no good reason to change it at the moment..

Reply:
ah sorry. that was my auto-formatter.